experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/shared/components.tsx

```diff
@@ -548,7 +558,11 @@ export const Footnotes: React.FC = () => {
       <FootnoteDivider></FootnoteDivider>
       <Footnote>
         <StyledMarker>*</StyledMarker>
-        {MAP_DISCLAIMER_TEXT}
+        <div>{MAP_DISCLAIMER_TEXT}</div>
+      </Footnote>
+      <Footnote>
+        <StyledMarker>*</StyledMarker>
```

Comment:
Perhaps we should update or remove the marker for the info text? Right now the `*` for the map disclaimer text works like a citation, each map chart has an asterisk to point to the disclaimer. Looks weird to have two `*` references.

Reply:
Done! updated screenshot
<img width="1607" alt="Screenshot 2023-09-12 at 4 55 19 PM" src="https://github.com/datacommonsorg/website/assets/13766/f1f02cf5-8066-493a-a5b6-ff66f4042145">